### PR TITLE
added gui on vagrant box

### LINKED ISSUE #60: LTSP server needs to have a GUI
LTSP server is used by the teacher, so it needs to have a GUI window. For example the teacher can use `epoptes` to check what the students are doing, or to show a demonstration to them.

### PR DESCRIPTION
**Pull request by d78ui98**
added gui

closes #60

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -62,3 +62,7 @@ fi
 
 # Restarting service
 service dnsmasq restart
+
+# installing and enabling gui
+apt --yes install xfce4 gtk2-engines-pixbuf
+startxfce4&
```
Comment (dashohoxha) [inline install.sh]: You should install `xubuntu-desktop`

Comment (d78ui98) [reply install.sh]: sure. I have tested it too.

**Review by dashohoxha — CHANGES_REQUESTED** [review]
I don't see the GUI added here.

**Comment by d78ui98** [comment]
please check here - https://github.com/docker-scripts/dev--LTSP/pull/65/commits/445a872f786594a020968b7ff1c235ea820e228b
Now that we are installing one package, I thought I will is better to add along with all the other installing packages.

**Review by dashohoxha — APPROVED** [review]
It seems Ok.

**Review by akash0x53 — APPROVED** [review]

**Pull request merged**